src/OnchainKitProvider.tsx

```diff
@@ -42,6 +42,11 @@ export function OnchainKitProvider({
           theme: config?.appearance?.theme ?? 'default',
         },
         paymaster: config?.paymaster || defaultPaymasterUrl,
+        wallet: {
+          display: config?.wallet?.display,
```

Comment:
should we just default to classic here since it seems like that's what we are doing in code?